include/libtorrent/ed25519.hpp

```diff
@@ -1,34 +0,0 @@
-#ifndef ED25519_HPP
```

Comment:
ah, deleting this file is what I believe is causing the documentation generation error. these functions used to be public and exported from libtorrent. now that this file can't be found anymore, there are no symbols that get categorized ed25519, and I think the file reference-ed25519 never gets generated